pkg/util/assert/assert.go

```diff
@@ -77,3 +82,108 @@ func asInt64(x any) (int64, bool) {
 
 	return 0, false
 }
+
```

Comment:
Using `callerInfo` seems like overkill to me.  And, I don't think we should take stuff some `testify`.  Useful for debugging, but now tests are passing ...

Reply:
I hear you.  But actually, this stuff is not going to fail once you've got it working.  You also won't necessarily need all of this to debug the problem.  In most cases it will be unnecessary.